From 3d5604ccdff750d760360f3c8bd8187226aa1a70 Mon Sep 17 00:00:00 2001
From: Ecco park <eccopark@broadcom.com>
Date: Wed, 13 Aug 2014 18:50:48 -0700
Subject: [PATCH 53/77] net: wireless: bcmdhd: change the policy of
 TCPACKSUPPRESS

Current policy(TCPACK_SUP_DELAYTX) is not verified fully.
So, we need to use TCPACK_SUP_REPLACE

16952284 dhd_tcpdata_info_get 930: No more free tdata_psh_info!!

Change-Id: Id721a409ca9411899fda9f89bdec6e9de6d7beda
Signed-off-by: Ecco park <eccopark@broadcom.com>
Signed-off-by: Dmitry Shmidt <dimitrysh@google.com>
---
 drivers/net/wireless/bcmdhd/dhd_linux.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/bcmdhd/dhd_linux.c b/drivers/net/wireless/bcmdhd/dhd_linux.c
index d8e2c3e98149..316adcafd2f1 100644
--- a/drivers/net/wireless/bcmdhd/dhd_linux.c
+++ b/drivers/net/wireless/bcmdhd/dhd_linux.c
@@ -4605,7 +4605,7 @@ dhd_attach(osl_t *osh, struct dhd_bus *bus, uint bus_hdrlen)
 #endif
 #ifdef DHDTCPACK_SUPPRESS
 #ifdef BCMSDIO
-	dhd_tcpack_suppress_set(&dhd->pub, TCPACK_SUP_DELAYTX);
+	dhd_tcpack_suppress_set(&dhd->pub, TCPACK_SUP_REPLACE);
 #elif defined(BCMPCIE)
 	dhd_tcpack_suppress_set(&dhd->pub, TCPACK_SUP_REPLACE);
 #else
-- 
2.0.3

